src/WPF/WPF.Viewer/Samples/Analysis/QueryFeatureCountAndExtent/QueryFeatureCountAndExtent.xaml.cs

```diff
@@ -68,39 +90,32 @@ private async Task Initialize()
             }
         }
 
-        private async void BtnZoomToFeaturesClick(object sender, RoutedEventArgs e)
+        private async void ZoomToFeatures(object sender, RoutedEventArgs e)
```

Comment:
See other samples, I think we should make it obvious that this is an event handler rather than a method we are calling directly. Is there a reason you've changed the name?